docker/initial-data.json

```diff
@@ -342,16 +340,19 @@
         "id": 4,
         "name": "Committees",
         "permissions": [
-            "agenda_item.can_see",
-            "agenda_item.can_see_internals",
+            "agenda_item.can_see_internal",
             "assignment.can_see",
             "meeting.can_see_frontpage",
             "projector.can_see",
             "mediafile.can_see",
             "motion.can_create",
-            "motion.can_see",
             "motion.can_support",
-            "users.can_see"
+            "user.can_see"
+            # Missing permissions compared with os3 and existing permissions in models.yml
+            "list_of_speakers.can_see",
+            "motion.can_create_amendments",
+            "user.can_see_extra_data",
```

Comment:
this is not included for committees

Reply:
Removed  "user.can_change_own_password"! Are you sure or is it just a remark for "user.can_see_extra_data"?

Reply:
Committees have `user.can_see` and `user.can_change_own_password`.

Reply:
Done